src/app/static/src/tests/components/reportLog/runningReportsList.test.ts

```diff
@@ -1,63 +1,61 @@
 import Vue from "vue";
 import {mount} from "@vue/test-utils";
 import RunningReportsList from "../../../js/components/reportLog/runningReportsList.vue";
+import VueSelect from "vue-select";
 
-function getWrapper(key: string = '') {
+const reports = [
```

Comment:
Pull reports out so that we can refer to elements in tests. Previously we selected by name but now we select by object.